css/index.css

```diff
@@ -0,0 +1,222 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  height: 100%;
```

Comment:
instead of using 100% for height, 100vh could be a better option

Reply:
- [ ] It would be a good example which I mentioned above to give a kind code review for students. Please consider fixing this one as well.